config/samples/resources/dnsrecordset/dns-routing-policy-primary-backup/compute_v1beta1_computesubnetwork_proxy.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 Google LLC
```

Comment:
This file should be "merged" into the previous one.

Here is how we group sample yamls: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/README.Samples.md#dependency-resources

```
Multiple dependencies of the same type should be defined in the same file, split by a YAML separation line (---).
```

Reply:
@anhdle-sso FYI